docker/images.json

```diff
@@ -1,15 +1,24 @@
 {
   "images": {
     "m3dbnode": {
+      "name": "m3dbnode",
       "dockerfile": "docker/m3dbnode/Dockerfile"
     },
+    "m3dbnode-setcap": {
+      "name": "m3dbnode",
+      "dockerfile": "docker/m3dbnode/Dockerfile",
+      "tag_suffix": "setcap"
```

Comment:
Will it add a "-"  automatically?  i.e. 0.10.2-setcap or will it be 0.10.2setcap ?

Reply:
Yeah, in the builder script we have
```shell
TAG="${TAG}-${TAG_SUFFIX}"
```

Reply:
Nice.